app/lib/actions/katello/capsule_content/refresh_repos.rb

```diff
@@ -0,0 +1,55 @@
+module Actions
+  module Katello
+    module CapsuleContent
+      module RefreshRepos
+        def fetch_proxy_service(_smart_proxy)
+          fail NotImplementedError
+        end
+
+        def act_on_repo?(_repo)
+          fail NotImplementedError
+        end
+
+        def plan(smart_proxy, options = {})
+          plan_self(:smart_proxy_id => smart_proxy.id,
+                    :environment_id => options[:environment]&.id,
+                    :content_view_id => options[:content_view_id]&.id,
+                    :repository_id => options[:repository]&.id)
+        end
```

Comment:
Works well for pulp3 Library repos per my testing. For CVs in other repos, I see this error: http://pastebin.test.redhat.com/787719 on capsule sync. 
Probably cause of this : `:content_view_id => options[:content_view_id]&.id,`

Reply:
Pulp2 yum works well for library env..For CV in other envs I see the above error for pulp2 and 3..